src/de/jost_net/JVerein/keys/KontoArt.java

```diff
@@ -0,0 +1,64 @@
+/**********************************************************************
+ * Copyright (c) by Heiner Jostkleigrewe
+ * This program is free software: you can redistribute it and/or modify it under the terms of the 
+ * GNU General Public License as published by the Free Software Foundation, either version 3 of the 
+ * License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,  but WITHOUT ANY WARRANTY; without 
+ *  even the implied warranty of  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See 
+ *  the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program.  If not, 
+ * see <http://www.gnu.org/licenses/>.
+ * 
+ * heiner@jverein.de
+ * www.jverein.de
+ **********************************************************************/
+package de.jost_net.JVerein.keys;
+
+public enum KontoArt
```

Comment:
Gibt es einen Grund, wieso hier der Name im Camel Case gewählt wurde und nicht wie bei anderen Arten ohne Binnen-Großbuchstabe?

Reply:
Es gibt genügend andere keys mit Großbuchstaben in der Mitte und ich glaube du hattest mir schon einmal einen Kommentar geschrieben der diese Schreibweise wollte.

Reply:
Habe es umbenannt

Reply:
> Es gibt genügend andere keys mit Großbuchstaben in der Mitte und ich glaube du hattest mir schon einmal einen Kommentar geschrieben der diese Schreibweise wollte.

Es ging darum, dass wir ja verschiedene Klassen haben die mit Art enden und bisher das Art nie großgeschrieben war. Danke fürs Ändern.